### PR TITLE
Texture Condition Checks

### PR DESCRIPTION
**Pull request by imKLM**
Allow for condition checks for specific textures (using fn_levelCheck). Works the same as vehicle condition checks.

#### Changes proposed in this pull request: 
- added condition check within fn_vehicleShopLBChange.sqf.
- added extra element into texture[] within Config_Vehicles.hpp (default "" for no check)

#### Example:
```
{ "Police lvl 3", "cop", {
    "texture\cop\offroad_rank3"
}, "call life_coplevel >= 3" }
```

### CODE REVIEW

File: Altis_Life.Altis/core/shops/fn_vehicleShopLBChange.sqf [hunk 1]
```diff
@@ -79,8 +79,13 @@ _colorArray = M_CONFIG(getArray,"LifeCfgVehicles",_classNameLife,"textures");
     _flag = (_x select 1);
     _textureName = (_x select 0);
     if ((life_veh_shop select 2) isEqualTo _flag) then {
-        _ctrl lbAdd _textureName;
-        _ctrl lbSetValue [(lbSize _ctrl)-1,_forEachIndex];
+        _x params["_texture"];
```
Comment (Jawshy) [inline Altis_Life.Altis/core/shops/fn_vehicleShopLBChange.sqf]: `params ["_texture"];`* 
Whitespace please.

File: Altis_Life.Altis/core/shops/fn_vehicleShopLBChange.sqf [hunk 2]
```diff
@@ -79,8 +79,13 @@ _colorArray = M_CONFIG(getArray,"LifeCfgVehicles",_classNameLife,"textures");
...
+        private _toShow = [_x] call life_fnc_levelCheck;
+
+        if (_toShow) then {
+            _ctrl lbAdd _textureName;
+            _ctrl lbSetValue [(lbSize _ctrl)-1,_forEachIndex];
```
Comment (Jawshy) [inline Altis_Life.Altis/core/shops/fn_vehicleShopLBChange.sqf]: *

**Comment by imKLM** [comment]
Changes made

**Comment by imKLM** [comment]
Added Apex Config changes (should work)

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 3]
```diff
@@ -1,12 +1,18 @@
 class CarShops {
     /*
+<<<<<<< HEAD
```
Comment (Jawshy) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: :open_mouth:

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 4]
```diff
@@ -1,12 +1,18 @@
...
+    *    ARRAY FORMAT:
+    *        0: STRING (Classname)
+    *        1: STRING (Condition)
+=======
```
Comment (Jawshy) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: :confounded:

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 5]
```diff
@@ -1,12 +1,18 @@
...
     *    FORMAT:
     *        STRING (Conditions) - Must return boolean :
     *            String can contain any amount of conditions, aslong as the entire
     *            string returns a boolean. This allows you to check any levels, licenses etc,
     *            in any combination. For example:
     *                "call life_coplevel && license_civ_someLicense"
     *            This will also let you call any other function.
+>>>>>>> refs/remotes/origin/master
```
Comment (Jawshy) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: :weary:

**Comment by imKLM** [comment]
made changes

**Comment by BoGuu** [comment]
LGTM - conflicts @imKLM

**Pull request merged**